### PR TITLE
[DOCS] Modifies schedule now transform API docs

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR fine-tunes the existing docs for the `shedule_now` endpoint.

### CODE REVIEW

File: docs/reference/transform/apis/schedule-now-transform.asciidoc [hunk 1]
```diff
@@ -1,14 +1,14 @@
 [role="xpack"]
 [testenv="basic"]
 [[schedule-now-transform]]
-= Schedule Now {transform} API
+= Schedule now {transform} API
 
 [subs="attributes"]
 ++++
-<titleabbrev>Shedule Now {transform}</titleabbrev>
+<titleabbrev>Shedule now {transform}</titleabbrev>
```
Comment (lcawl) [inline docs/reference/transform/apis/schedule-now-transform.asciidoc:8]:
```suggestion
<titleabbrev>Schedule now {transform}</titleabbrev>
```

File: docs/reference/transform/apis/schedule-now-transform.asciidoc [hunk 2]
```diff
@@ -24,10 +24,10 @@ in the `transform_admin` built-in role.
 [schedule-now-transform-desc]]
 == {api-description-title}
 
-If you _schedule_now a {transform}, it will process the new data instantly,
+If you `_schedule_now` a {transform}, it will process the new data instantly,
```
Comment (lcawl) [inline docs/reference/transform/apis/schedule-now-transform.asciidoc:27]: IMO the use of the endpoint name as a verb is unwieldy. I think it's simpler like this:
```suggestion
When you run this API, the {transform} processes the new data instantly,
```

File: docs/reference/transform/apis/schedule-now-transform.asciidoc [hunk 3]
```diff
@@ -24,10 +24,10 @@ in the `transform_admin` built-in role.
...
 without waiting for the configured `frequency` interval.
-After _schedule_now API is called, the transform will be processed again at
-`now + frequency` unless _schedule_now API is called again in the meantime.
+After the _schedule_now API is called, the transform will be processed again at
```
Comment (lcawl) [inline docs/reference/transform/apis/schedule-now-transform.asciidoc:29]: Likewise, I think this is simpler:
```suggestion
Subsequently, the transform will be processed again at
```

**Review by przemekwitek — COMMENTED** [review]
LGTM

Thanks!

**Pull request merged**